Assignment/solution_student/index.html

```diff
@@ -1,35 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+    <title>Joe Johnson</title>
+    <link rel="stylesheet" href="style.css">
+</head>
 
-Sample Content
-——-
+<header>
 
-ABOUT ME PAGE
+<h2><em>About Me</em> | <a href="resume.html">Porfolio</a></h2>
+<h1>Joe Johnson</h1>
 
-I ENJOY LIFE AS A DEVELOPER
 
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
+</header>
 
+<body>
```

Comment:
Ooops! The `<body>` tag contains **all** of the content of the site, remember? So the opening `<body>` should be the very first thing you write after the closing `</head>` — you might have gotten the `<header>` mixed up with the `<head>`, but just remember the two _are_ different.